code/_onclick/hud/radial.dm

```diff
@@ -312,4 +312,22 @@ var/global/list/radial_menus = list()
 	radial_menus -= uniqueid
 	return answer
 
-#define RADIAL_INPUT(user, choices) show_radial_menu(user, user, choices)
+/proc/get_mob_head_mutable_appearance(var/mob/M)
+	var/mutable_appearance/MA = mutable_appearance(M.icon, M.icon_state)
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(H.head?.flags_inv & HIDEFACE)
+			MA = mutable_appearance(H.head.icon, H.head.icon_state)
+		else if(H.wear_mask?.flags_inv & HIDEFACE)
+			MA = mutable_appearance(H.wear_mask.icon, H.wear_mask.icon_state)
+		else
+			var/obj/item/organ/external/head/head = H.organs_by_name[BP_HEAD]
+			MA = mutable_appearance(head.icon, head.icon_state)
```

Comment:
I am 99% sure that this does nothing, because heads don't have an icon or state

Reply:
it worked in testing before i added the overlay

Reply:
bruh